doc/classes/SpringArm.xml

```diff
@@ -33,17 +41,25 @@
 			<argument index="0" name="RID" type="RID">
 			</argument>
 			<description>
+			Remove the given objects from the objects excluded from the collision check
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="collision_mask" type="int" setter="set_collision_mask" getter="get_collision_mask" default="1">
+			The layers against which the collision check shall be done.
 		</member>
 		<member name="margin" type="float" setter="set_margin" getter="get_margin" default="0.01">
+			When the collision check is made, a candidate length for the SpringArm is given.
+			The margin is then subtracted to this length and the translation is applied to the child objects of the SpringArm.
+			This margin is useful for when the SpringArm has a Camera as a child node: without the margin, the Camera would be placed on the exact point of collision, while with the margin the Camera would be placed close to the point of collision.
```

Comment:
Use `[Camera]` to make it an hyperlink to the Camera docs. (You can also use `[SpringArm]` btw in this file too, instead of making a hyperlink to self it will automatically be formatted as inline code.)